tests/factories/cht/generate.js

```diff
@@ -51,7 +60,63 @@ const createData = ({ healthCenter, user, nbrClinics=10, nbrPersons=10 }) => {
   return { clinics, reports, persons };
 };
 
-const createHierarchy = ({ name, user=false, nbrClinics=50, nbrPersons=10 }) => {
+const createDataWithRealNames = ({ healthCenter, user, nbrClinics = 10, nbrPersons = 10 }) => {
+  const clinics = Array.from({ length: nbrClinics }).map((_, index) => {
```

Comment:
It's better to avoid big callbacks on functions and try to keep code as flat as possible. Can you try to make it a function so it's something similar to this? 
```js
const clinics = Array
  .from({ length: nbrClinics })
  .map((_, index) => createClinic(index));
```